website/src/controllers/api-v2/unsubscription.js

```diff
@@ -15,7 +15,7 @@ api.unsubscribe = function(req, res, next){
       $set: {'preferences.emailNotifications.unsubscribeFromAll': true}
     }, {multi: false}, function(err, updateRes){
       if(err) return next(err);
-      if(updateRes !== 1) return res.status(404).json({err: 'User not found'});
+      if(updateRes.n !== 1) return res.status(404).json({err: 'User not found'});
```

Comment:
This is okay, the change comes from the mongodb native driver for node not from mongodb 3